content/docs/latest/installing/bare-metal/raspberry-pi.md

```diff
@@ -103,9 +103,60 @@ chmod +x flatcar-install
 mv flatcar-install ~/.local/bin
 ```
 
+##### Set up Ignition config
+
+Once the `flatcar-install` script has been installed, we can begin to write a Butane configuration file that can then be transpiled to Ignition.
+
+[Why Butane](butane-transpiler)? The Butane configuration transpiler reads from an easier-to-write language (YAML), and parses it through to check for errors. This means that if you write a valid Butane configuration, and it successfully transpiles to JSON, you are guaranteed to have a valid Ignition configuration.
+
+To generate your Ignition config, first start with the following `butane-config.yml`:
+
+```yaml
+variant: flatcar
+version: 1.0.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - <paste authorized SSH keys here>
+
+
+kernel_arguments:
+  should_exist:
+    - console=tty0 # Used to tell the Linux kernel to output console to its display adapter
+  should_not_exist:
+    - quiet # Show logs upon kernel boot
+    - console=ttyAMA0,115200n8 # Disable the console being output via serial on the Raspberry Pi, so HATs can utilize that UART device.
+
+storage:
+  files:
+    - path: /oem/grub.cfg
+      contents:
+        inline: set linux_console="console=tty0" # Create a grub.cfg that contains appropriate arguments for the linux command line. In this case, we are telling it to use tty0 for the console
+      mode: 0644
+      overwrite: true # There may be an existing file there, we want to overwrite it
```

Comment:
This is wrong if you want to use `kernel_arguments` because it overwrites them again. Decide for one approach and if one would have to split it up due to console vs other args then one would need an 'append' section.